File: programming languages/28. prova esame/Tennis.java
public class Tennis extends Campo {
    float larghezza, lunghezza, tempMedia, costo;
    String terreno;

    public Tennis(int codice, String nome, float larghezza, float lunghezza, float tempMedia, float costo, String terreno) {
        super(codice, nome);
        this.larghezza = larghezza;
        this.lunghezza = lunghezza;
        this.tempMedia = tempMedia;
        this.costo = costo;
        this.terreno = terreno;
    }

    public float getLarghezza() {
        return larghezza;
    }

    public float getLunghezza() {
        return lunghezza;
    }

    public float getTempMedia() {
        return tempMedia;
    }

    public float getCosto() {
        return costo;
    }

    public String getTerreno() {
        return terreno;
    }

    @Override
    public String toString() {
        return "Tennis" + "\t" + getNome() + "\t" + getCodice() + "\t" + getLarghezza() + "\t" + getTempMedia() + "\t" + getTerreno() + "\t-\t-\t" + getCosto();
    }
}

File: Fondamenti_programmazione_II/Python/Esami/Gestione_circolo_sportivo/Tennis.py
from Campo import Campo

class Tennis(Campo):
    def __init__(self, codice, campo, larghezza, lunghezza, temperatura, terreno, costo):
        super().__init__(codice,campo, larghezza, lunghezza, costo, "tennis")
        self._temperatura = temperatura
        self.terreno = terreno

    def __str__(self):
        return super().__str__() + str(self._larghezza) +"\t"+ str(self._lunghezza) +"\t"+ str(self._temperatura ) +"\t"+ str(self.terreno) +"\t-\t-\t"+ str(self._costo)